RFCs/0018-Skip-Default-Extensions.md

```diff
@@ -0,0 +1,34 @@
+# 0018 Skip Default Extensions
+
+## Summary
+This note outlines the proposed changes for adding option for skipping lookup and initialization of default extensions.
+
+## Motivation
+In addition to test adapters provided via TestAdaptersPaths, test platform requires default adapters present in Extensions folder. This folder lies where vstest.console.exe lies. Even if default extensions can't run/discover tests, we still pass test assembly to default extensions and thus increase adapter lookup and initialization cost. 
+
+Example: Default extensions can't run/discover XUnit tests, still we initialize default adapters and pass test assembly to them. Thus we   
+
+## Proposed Changes
+1. Design mode clients can skip default extensions by setting TestPlatformOptions.SkipDefaultExtensions as true.
```

Comment:
Some of these are internal details. We should have not have internal details of implementation in RFC.

Reply:
Only the APIs change (including internal APIs) are mentioned here.